### DISCUSSION TITLE
Welcome to DLS Official Web Project Discussions!

### DISCUSSION THREAD

**Discussion by kasuncfdo** [kasuncfdo/dls#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kasuncfdo** [comment]
### **<p align="center"> Navigathion Schema </p>**
<p align="center">
  <img width="200" height="200"src="https://raw.githubusercontent.com/kasuncfdo/dls/main/College%20Assests/College-Items/DLS_WEBCREST.png">

## Yellows
[ View ](https://www.delasalle.lk) `Home` - DLS

**Reply by pauloiys** [reply]
😊👏👏

**Reply by kasuncfdo** [reply]
> 😊👏👏

(●'◡'●)
